### DISCUSSION TITLE
Bring to front / focus on window once scanning is finished

### DISCUSSION THREAD

**Discussion by mineminemine** [Andrewthe13th/Inventory_Kamera#353] [Ideas or Feature requests]
Currently, the scanner does nothing once it has finished scanning and will stay in the background while the game is still in focus. I think it's imperative that the scanner window gets focused once the scanning is completed so as to show that it has completed the scanning instead of doing nothing, which could potentially be confusing to those that don't realize the scanner has finished its task.

**Comment by Cupcak3** [comment]
This has been implemented in V1.3.0
